12.LOCO_BULK_MLP/0.data_splits/scripts/0.LOCO_data_split.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import argparse
+import pathlib
+
+import pandas as pd
+import toml
+
+# In[2]:
+
+
+argparser = argparse.ArgumentParser()
+argparser.add_argument("--cell_type", default="all")
+
+args = argparser.parse_args()
+
+cell_type = args.cell_type
```

Comment:
Is this to run each cell type in the bash script? Weren't you using `papermill` (might be the wrong name) to perform this? I could be forgetting how it works lol.

Reply:
I was, but I am electing to use scripts to run as they are more compatible with ALPINE should I need to use it. :D